src/main/java/org/mule/module/apikit/api/deserializing/ArrayHeaderDelimiter.java

```diff
@@ -11,15 +11,19 @@
  */
 public enum ArrayHeaderDelimiter {
 
-  COMMA(",");
+  COMMA(',');
 
-  private final String delimiter;
+  private final char delimiter;
 
-  ArrayHeaderDelimiter(String delimiter) {
+  ArrayHeaderDelimiter(char delimiter) {
```

Comment:
Considering that this is API, I think we should keep the old constructor as well

Reply:
You are right! I missed that slight and important detail 😓